homeassistant/components/schlage/quality_scale.yaml

```diff
@@ -0,0 +1,84 @@
+rules:
+  # Bronze
+  config-flow: done
+  test-before-configure: done
+  unique-config-entry: done
+  config-flow-test-coverage: done
+  runtime-data: done
+  test-before-setup: done
+  appropriate-polling: done
+  entity-unique-id: done
+  has-entity-name: done
+  entity-event-setup: done
+  dependency-transparency: done
+  action-setup:
+    status: exempt
+    comment: This integration does not have any actions.
+  common-modules: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  docs-actions:
+    status: exempt
+    comment: This integration does not have any actions.
+  brands: done
+
+  # Silver
+  config-entry-unloading: done
+  log-when-unavailable: done
+  entity-unavailable: done
+  action-exceptions:
+    status: exempt
+    comment: This integration does not have any actions.
+  reauthentication-flow: done
+  parallel-updates:
+    status: done
+    comment: Implicitly handled by DataUpdateCoordinator
+  test-coverage: done
+  integration-owner: done
+  docs-installation-parameters:
+    status: exempt
+    comment: There are no installation parameters.
+  docs-configuration-parameters:
+    status: exempt
+    comment: This integration does not utilize an options flow.
+
+  # Gold
+  entity-translations: done
+  entity-device-class: done
+  devices: done
+  entity-category: done
+  entity-disabled-by-default: done
+  discovery:
+    status: exempt
+    comment: Schlage devices cannot be auto-discovered
```

Comment:
No mDNS packets? Did you check if Schlage has an OUI so you can use the mac address to find the device on the network?